packages/dice-webapp/components/sidebar/sidebar-data.tsx

```diff
@@ -0,0 +1,63 @@
+import { LuLayoutDashboard } from "react-icons/lu";
+import {
+  BiSolidDashboard,
+  BiMessageSquareError,
+  BiSolidMessageSquareError,
+} from "react-icons/bi";
+import {
+  PiCertificate,
+  PiCertificateFill,
+  PiUserBold,
+  PiUserFill,
+} from "react-icons/pi";
+import {
+  AiOutlineSafetyCertificate,
+  AiFillSafetyCertificate,
+} from "react-icons/ai";
+import { PiClockCountdown, PiClockCountdownFill } from "react-icons/pi";
```

Comment:
Group these imports from react-icons/pi with the above import from react-icons/pi